source/Halibut.Tests/CancellationViaClientProxyFixture.cs

```diff
@@ -52,9 +52,11 @@ public async Task CancellationCanBeDoneViaClientProxy(ClientAndServiceTestCase c
         }
 
         [Test]
-        public async Task CannotHaveServiceWithHalibutProxyRequestOptions()
+        [LatestClientAndLatestServiceTestCases(testNetworkConditions: false, testWebSocket: false, testPolling:false)]
```

Comment:
To specifically get just the "Listening" test cases, we had to turn off everything else.

We have done it this way throughout these tests.

I wonder if there is a trap here for the future, because if we add another type, then these tests will start running those as test cases.

Maybe we should look at making it easier to specify "inclusions" rather than "exclusions"?

Reply:
Leave as is to force people to review existing tests, safety first!